cmd/tink-cli/cmd/hardware/get.go

```diff
@@ -57,11 +57,19 @@ func (h *getHardware) PopulateTable(data []interface{}, t table.Writer) error {
 				t.AppendRow(table.Row{
 					hw.Id,
 					iface.Dhcp.Mac,
-					iface.Dhcp.Ip.Address,
+					getIpAddressValForAppendRow(iface.Dhcp.Ip),
 					iface.Dhcp.Hostname,
 				})
 			}
 		}
 	}
 	return nil
 }
+
+func getIpAddressValForAppendRow(ip *hwpb.Hardware_DHCP_IP) string {
```

Comment:
Please change the name of the function to a bit short like getIpAddress or something more suitable but short.